src/components/Modal/Modal.css

```diff
@@ -0,0 +1,47 @@
+.modal-container {
+  /* display: none; */
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 99999999;
```

Comment:
you should not use such high z-index

Reply:
Thanks Anuraj for pointing that out. I have changed it a lower value.